backend/hitas/services/condition_of_sale.py

```diff
@@ -0,0 +1,122 @@
+from django.conf import settings
+from django.db import models
+from django.db.models import Q
+from django.utils import timezone
+
+from hitas.models import ConditionOfSale, Owner, Ownership
+from hitas.models.condition_of_sale import ConditionOfSaleAnnotated
+from hitas.services.apartment import subquery_first_purchase_date
+
+
+def condition_of_sale_queryset() -> models.QuerySet[ConditionOfSaleAnnotated]:
```

Comment:
Refactoring code should be in their own commits, not mixed in the same commit with unrelated changes.

Reply:
I refactored these because I initially had the apartment and condition of sale functions in the respective model's files, but that caused import loops, so I had to move both of them to a new place.